resources/model/OpenStudio.idd

```diff
@@ -21266,6 +21266,21 @@ OS:Controller:OutdoorAir,
        \default BypassWhenWithinEconomizerLimits
        \key BypassWhenWithinEconomizerLimits
        \key BypassWhenOAFlowGreaterThanMinimum
+  A21; \field Economizer Operation Staging
+       \type choice
+       \key EconomizerFirst
+       \key InterlockedWithMechanicalCooling
+       \default InterlockedWithMechanicalCooling
```

Comment:
As usual, torn on whether we should make this required-field or not... The field above isn't.

@kbenne

Reply:
I think because it is a new field we don't risk breaking an API so required field is better. That is, fewer options is better and I don't see much downside except perhaps being a little inconsistent with similar cases.